spec/services/lara_serialization_helper_spec.rb

```diff
@@ -84,6 +122,48 @@
         expect(result.interactive).to eq interactive
       end
     end
+
+    describe "import page with interactives linked to other interactives" do
+      let(:export_data) do
+        {type: "InteractivePage", ref_id: "100-InteractivePage", embeddables: [
+          {embeddable: {type: "MwInteractive", ref_id: "100-MwInteractive", linked_page_items: [{:primary_ref_id=>"100-MwInteractive", :secondary_ref_id=>"101-ManagedInteractive", :label=>"one"}, {:primary_ref_id=>"100-MwInteractive", :secondary_ref_id=>"102-MwInteractive", :label=>"two"}]}, section: "interactive_box"},
+          {embeddable: {type: "ManagedInteractive", ref_id: "101-ManagedInteractive", linked_page_items: []}, section: "interactive_box"},
+          {embeddable: {type: "MwInteractive", ref_id: "102-MwInteractive", linked_page_items: []}, section: "interactive_box"},
+          {embeddable: {type: "ManagedInteractive", ref_id: "103-ManagedInteractive", linked_page_items: [{:primary_ref_id=>"103-ManagedInteractive", :secondary_ref_id=>"100-MwInteractive", :label=>"three"}]}, section: "interactive_box"}
+        ]}
```

Comment:
Can you clean this up a bit?

Better indenting would be helpful and not mixing `key:` with `:key=?` would also help.